### ISSUE TITLE
Allow unauthenticated read-only access to notes

### ISSUE THREAD

**Issue by alansmitheejp** [zadam/trilium#391]
I am happy if anyone can read it with Web access.

**Comment by zadam** [comment]
Can you clarify what you mean?

**Comment by alansmitheejp** [comment]
I want to publish the knowledge database on the web.
Anyone can browse contents from web browser without user ID.
Only me is registering knowledge.

**Comment by zadam** [comment]
This is currently not possible although I understand the use case. I think instead of trying to incorporate this feature into Trilium itself it would make more sense to create a separate read-only unauthenticated web app (possibly as a sync server).

**Comment by thehotery** [comment]
> think instead of trying to incorporate this feature into Trilium itself it would make more sense to create a separate read-only unauthenticated web app (possibly as a sync server).

it will be great for that to have feature to export notes in markdown format, so it can be used in any tools that support it (like Jekyll, Pelican and other)...

**Comment by zadam** [comment]
Not sure if I understand but markdown as an export format is already supported ...

**Comment by thehotery** [comment]
some kind of automated (by schedule) export, for example, every day; or, if possible, to invoke it (export) from command line/script

**Comment by Mangiola** [comment]
Recently I've tried creating this kind of read-only view in a very hacky way. I exported my notes to a new Trilium instance and made all notes inherit #readOnly. I also removed most UI elements with CSS that were related to editing notes.

This is clearly flawed, as there is still functionality that cannot be disabled (like moving notes in the tree). Exporting all my notes to another instance is also a hassle. I'm currently looking into other solutions, as this would be a very useful feature for me.
